corehq/apps/app_manager/migrations/0018_migrate_case_search_labels.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.toggles import SYNC_SEARCH_CASE_CLAIM
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _migrate_case_search_labels(apps, schema_editor):
+    for domain in sorted(SYNC_SEARCH_CASE_CLAIM.get_enabled_domains()):
+        call_command('migrate_case_search_labels', domain=domain)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app_manager', '0017_migrate_case_search_relevant'),
+    ]
+
+    operations = [
+        migrations.RunPython(_migrate_case_search_labels,
+                             reverse_code=migrations.RunPython.noop,
+                             elidable=True),
```

Comment:
would be nice if you could use `run_once_off_migration` here instead of doing it manually. That would require changing the management command to allow passing all the enabled domains at once instead of individually.

Reply:
This would need changes to the base management command, `AppMigrationCommandBase`.

Are there advantages to using that here?

From the documentation
```
This will give the user directions for running the command
    manually if the command fails or has since been removed from the codebase
```

We probably can remove the management command once its deployed to all environments or in a certain future, or possibly get rid of the migration and the command both once this is deployed on all envs.

Reply:
The trouble is supporting this for 3rd part envs. We could indeed just delete this after some period of time though I don't know what that period is.

Reply:
This is behind a feature flag `USH_CASE_CLAIM_UPDATES` if that makes any difference.
I am leaning towards keeping it as it is. Let me know if there is a blocking change @snopoke

Reply:
To be particular, this isn't exactly behind `USH_CASE_CLAIM_UPDATES`, it'll also affect domains with `SYNC_SEARCH_CASE_CLAIM` which is a more widely-used flag, though I don't know if there are local hosters using it.

Regarding timeline if we later delete this, we recommend local hosters deploy every two weeks: http://dimagi.github.io/commcare-cloud/system/maintenance-expectations.html

Reply:
> Regarding timeline if we later delete this, we recommend local hosters deploy every two weeks: http://dimagi.github.io/commcare-cloud/system/maintenance-expectations.html

I think there's a difference between what we recommend for deploy and what we 'support' for keeping migrations working. FYI this is already on the agenda for the next cross-divisional meeting.

Reply:
@orangejenny thanks for pointing that out.

It is because of this I thought this was the USH feature flag. It's only visible on the UI if this feature flag is enabled.

https://github.com/dimagi/commcare-hq/blob/d58656b0262958a8c5f4c98718c9549201e6ff29/corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html#L111-L132

Reply:
> I think there's a difference between what we recommend for deploy and what we 'support' for keeping migrations working. FYI this is already on the agenda for the next cross-divisional meeting

So if we remove the migration from code, the migrations otherwise would still work?

Would be good to finalize on this, there is at least one more management command `migrate_case_search_relevant` + its migration, added just before the one added in this PR, which we can remove as well, if it is considered okay to remove

Reply:
@mkangia I see what you mean about the flags. For historical context (which I had forgotten until you pointed this out): the `command_label` attribute was part of the original case claim feature, but it wasn't originally visible in the UI. USH made the changes to add it to the UI, make it translate-able, and also added the "Search Again" workflow. So the labels are only visible in the UI if the the USH feature flag is on, but app docs can still have the `command_label` attribute even if they use the regular case claim flag not the USH flag. Plus, the USH flag was added some time after the USH changes were made.

Reply:
Woops thanks. I will update the PR description to include both feature flags.
can you confirm if there is anything that needs to be handled differently as far as this PR is concerned?

Also,
If it was not in the UI, how was it set? via backend? and will there will many domains that would have that set but not have the USH specific feature flag enabled?

Reply:
The code in this PR looks fine regarding flags.

> If it was not in the UI, how was it set? via backend? and will there will many domains that would have that set but not have the USH specific feature flag enabled?

Prior to USH putting it in the UI, apps would have had the `command_label` attribute, but it would always just be set to the default value, they had no way to change it. Then USH put it in the UI, under the regular search & claim flag. Then a month or two later, USH moved the UI behind the USH flag. So the only domains that aren't using the USH flag and have a non-default value would be any that happened to see the UI and change the label before it was switched to the USH flag. I doubt that's many, if any, domains.

Reply:
sweet, thanks.

Reply:
Fine to leave the migration as is. Can always be changed at a later point.